lib/Plans.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+// Copyright 2022. Plesk International GmbH. All rights reserved.
+
+namespace WHMCS\Module\Server\Plesk360Monitoring;
+
+use Exception;
+use WHMCS\Module\Server\Plesk360Monitoring\Plans\BusinessPlan;
+use WHMCS\Module\Server\Plesk360Monitoring\Plans\EnterprisePlan;
+use WHMCS\Module\Server\Plesk360Monitoring\Plans\FlexiblePlan;
+use WHMCS\Module\Server\Plesk360Monitoring\Plans\ProPlan;
+
+class Plans
+{
+    /**
+     * @return Plan[]
+     */
+    public static function getAll(): array
```

Comment:
plans can be initialzed in the constructor and kept in property in an array with the id as a key

Reply:
Done